tests/unit/models/reminder-test.js

```diff
@@ -0,0 +1,12 @@
+import { moduleForModel, test } from 'ember-qunit';
+
+moduleForModel('reminder', 'Unit | Model | reminder', {
+  // Specify the other units that are required for this test.
+  needs: []
+});
```

Comment:
Take this test out of you're not going to use it. I'm only expecting you to write acceptance tests